### PR TITLE
Added arequal-checksum install script

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Added the script and also the logic to install the script if not present
in the machines.

Fixes: #883

Signed-off-by: Ayush Ujjwal <aujjwal@redhat.com>

### CODE REVIEW

**Comment by aujjwal-redhat** [comment]
```log
[2021-07-27 13:34:36,496] INFO  - Running arequal-checksum on 10.70.43.23
[2021-07-27 13:34:36,536] DEBUG  - {'Flag': False, 'msg': [], 'error_msg': "Usage: arequal-checksum [OPTION...] \nTry `arequal-checksum --help' or `arequal-checksum --usage' for more\ninformation.\n", 'node': '10.70.43.23', 'cmd': 'arequal-checksum', 'error_code': 64}
[2021-07-27 13:34:36,536] INFO  - Running arequal-checksum on 10.70.41.159
[2021-07-27 13:34:36,574] DEBUG  - {'Flag': False, 'msg': [], 'error_msg': "Usage: arequal-checksum [OPTION...] \nTry `arequal-checksum --help' or `arequal-checksum --usage' for more\ninformation.\n", 'node': '10.70.41.159', 'cmd': 'arequal-checksum', 'error_code': 64}
[2021-07-27 13:34:36,577] INFO  - Running arequal-checksum on 10.70.43.115
[2021-07-27 13:34:36,617] DEBUG  - {'Flag': False, 'msg': [], 'error_msg': "Usage: arequal-checksum [OPTION...] \nTry `arequal-checksum --help' or `arequal-checksum --usage' for more\ninformation.\n", 'node': '10.70.43.115', 'cmd': 'arequal-checksum', 'error_code': 64}
[2021-07-27 13:34:36,619] INFO  - Running arequal-checksum on 10.70.43.74
[2021-07-27 13:34:36,658] DEBUG  - {'Flag': False, 'msg': [], 'error_msg': "Usage: arequal-checksum [OPTION...] \nTry `arequal-checksum --help' or `arequal-checksum --usage' for more\ninformation.\n", 'node': '10.70.43.74', 'cmd': 'arequal-checksum', 'error_code': 64}
[2021-07-27 13:34:36,660] INFO  - Creating thread queues for the tests
[2021-07-27 13:34:36,672] INFO  - Starting Disruptive test case runs.
[2021-07-27 13:34:37,625] INFO  - Running test_user_ops-rep
```

I changed the error code condition to see if the script is running and it worked fine..

File: tools/arequal_install.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+mkdir /tmp/arequal_repo
```
Comment (srijan-sivakumar) [inline tools/arequal_install.sh:1]: best practice to add the shell interpreter env at the beginning.

#!/bin/sh, now we can keep that as #!/bin/bash but for portability between different shells ( we don't know if people would be using BASH or ZSH or KSH ), so better to use #!/bin/sh.

**Review by srijan-sivakumar — CHANGES_REQUESTED** [review]
Do check the

File: tools/arequal_install.sh (hunk 1 above)
Comment (aujjwal-redhat) [reply tools/arequal_install.sh:1]: ack

**Review by srijan-sivakumar — APPROVED** [review]

**Pull request merged**